examples/static-preload.html

```diff
@@ -0,0 +1,17 @@
+<head>
+<link rel=preload as=script href=https://securepubads.g.doubleclick.net/tag/js/gpt.js>
+</head>
+
+<body>
+<script>
+  // Busy loop to block the load event.
+  const then = performance.now();
+  while (performance.now() - then < 2000);
```

Comment:
Even though the comment is there, I think this would be clearer if the semicolon were on the following line.